java/group-layers/src/main/java/com/esri/arcgisruntime/sample/grouplayers/MainActivity.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2019 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.sample.grouplayers;
+
+import java.util.Arrays;
+import java.util.List;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.BottomSheetBehavior;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.RecyclerView;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.MotionEvent;
+import android.view.View;
+
+import com.esri.arcgisruntime.data.ServiceFeatureTable;
+import com.esri.arcgisruntime.layers.ArcGISSceneLayer;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.layers.GroupLayer;
+import com.esri.arcgisruntime.layers.Layer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.DefaultSceneViewOnTouchListener;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.samples.grouplayers.R;
+import com.esri.arcgisruntime.util.ListenableList;
+
+public class MainActivity extends AppCompatActivity implements OnLayerCheckedChangedListener {
+
+  private SceneView mSceneView;
+  private BottomSheetBehavior mBottomSheetBehavior;
+  private RecyclerView mLayersRecyclerView;
+  private LayersAdapter mLayersAdapter;
+
+  @Override protected void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.activity_main);
+
+    mSceneView = findViewById(R.id.sceneView);
+    mLayersRecyclerView = findViewById(R.id.layersRecyclerView);
+
+    View bottomSheet = findViewById(R.id.bottomSheet);
+    mBottomSheetBehavior = BottomSheetBehavior.from(bottomSheet);
+    mBottomSheetBehavior.setState(BottomSheetBehavior.STATE_HIDDEN);
+
```

Comment:
The top fix here seemed to solve the button not working for me: https://stackoverflow.com/questions/37822264/android-bottom-sheet-behavior-not-working-properly-views-not-show-on-first-run

It also seems to correlate with the scene view's attribution text not working.

Want to explore this a bit?

Reply:
Fixed. Thanks @TADraeseke